packages/ruleset/test/property-attributes.test.js

```diff
@@ -145,11 +145,13 @@ describe('Spectral rule: property-attributes', () => {
         };
 
         const results = await testRule(ruleId, rule, testDocument);
-        expect(results).toHaveLength(3);
```

Comment:
this testcase needed to be updated because of changes I made to the rootDocument testcase